pkg/proxyserver/apis/clusterview/v1alpha1/register.go

```diff
@@ -0,0 +1,34 @@
+package v1alpha1
+
+import (
+	clusterv1alpha1 "github.com/open-cluster-management/api/cluster/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const GroupName = "clusterview.open-cluster-management.io"
+
+var (
+	GroupVersion  = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
+	schemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+	// Install is a function which adds this version to a scheme
+	Install = schemeBuilder.AddToScheme
+
+	// SchemeGroupVersion generated code relies on this name
+	// Deprecated
```

Comment:
The function seems to be deprecated since its first day.. is this a typo? @zhiweiyin318